server/plugin/create.go

```diff
@@ -186,7 +186,7 @@ func (p *Plugin) handleCreate(c *plugin.Context, args *model.CommandArgs, parame
 		Dialog: model.Dialog{
 			CallbackId:     fmt.Sprintf("create_%s", subcommand),
 			Title:          fmt.Sprintf("Create a Google %s", cases.Title(language.English, cases.NoLower).String(subcommand)),
-			IconURL:        "http://www.mattermost.org/wp-content/uploads/2016/04/icon.png",
+			IconURL:        "https://www.mattermost.org/wp-content/uploads/2016/04/icon.png",
```

Comment:
Should this be a Google Drive icon instead?
Isn't it better to bundle this icon in the plugin in case that file gets removed?

Reply:
@fmartingr I was wondering the same thing about it being bundled in the plugin as well.
As for which image it should be, I'm not 100% sure. It was a quick fix so I figured I would do it myself.
Maybe this warrants a ticket. @BenCookie95?

Reply:
Yeah we should probably bundle that in. I'll create a separate ticket